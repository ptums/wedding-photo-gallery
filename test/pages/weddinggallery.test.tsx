import React from 'react'
import { render } from '@testing-library/react'
import WeddingGallery from '../../pages/wedding-gallery'

describe('Wedding Gallery', () => {
  const userAgent = {
    os: 'Android',
    deviceType: 'mobile',
  }
  it('matches snapshot', () => {
    const { asFragment } = render(<WeddingGallery userAgent={userAgent} />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
