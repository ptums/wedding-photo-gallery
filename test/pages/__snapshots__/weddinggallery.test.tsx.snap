// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wedding Gallery matches snapshot 1`] = `
<DocumentFragment>
  <main
    class="jsx-2359628166"
  />
</DocumentFragment>
`;
